pom.xml

```diff
@@ -238,6 +238,8 @@
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
     <error-prone.configuration-args>
+      <!-- `JsonCreator.Mode` should be set for single-argument creators. -->
+      -Xep:AmbiguousJsonCreator:ERROR
```

Comment:
Do we really need to write a reason for each type of check we are turning on? From the main issue this will be a long list, and we don't even document our own checks on why we are turning them on. We only do this if we are turning them off.

Reply:
This is completely up to your preference 😄. I'm fine with both options, we can simply omit it.

Reply:
@romani

Reply:
I vote to skip comment.
We do comment only when we disable rule , OFF .

Reply:
Deal, makes sense. Will update!